src/deploy.sh

```diff
@@ -152,10 +152,29 @@ function kernel_deploy()
 #
 function deploy_parser_options()
```

Comment:
we discussed this a while a go, but what do you think about taking this opportunity to rename this function to `parse_deploy_options`?